content/en/docs/reference/topics-on-dockershim-and-cri-compatible-runtimes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Topics on dockershim and CRI-compatible runtimes
+content_type: reference
+weight: 20
+---
+<!-- overview -->
+Articles about the Kubernetes' deprecation of dockershim, and using CRI-compatible runtimes.
```

Comment:
```suggestion
This is a list of articles about:
- the Kubernetes' deprecation and removal of _dockershim_
- using CRI-compatible container runtimes
```

The key part for me is “and removal”. I want readers who aren't clear on the meaning of “deprecation” to infer that it's something that happens before removal.

Reply:
Good point! Updated. Thanks.